### ISSUE TITLE
util.timezone init error when copy() (python2 only)

### ISSUE THREAD

**Issue by luckylud** [wbond/asn1crypto#167]
With python 2 when calling core.Asn1Value.untag()

> .../asn1crypto/core.py", line 568, in untag
>     new_obj._copy(self, copy.deepcopy)
>   File ".../asn1crypto/core.py", line 4524, in _copy
>     super(SequenceOf, self)._copy(other, copy_func)
>   File ".../asn1crypto/core.py", line 593, in _copy
>     self._native = copy_func(other._native)
> .....
>   File "/usr/lib/python2.7/copy.py", line 329, in _reconstruct
>     y = callable(*args)
> TypeError: __init__() takes at least 2 arguments (1 given)
> 

In asn1crypto.util
Set default timezone offset with timedelta(0) fix it:
`def __init__(self, offset=timedelta(0), name=None):`

**Comment by joernheissler** [comment]
Thanks for the report!
Could you add a small testcase please? I don't know how to reproduce the problem.

Python3's timezone constructor has no default value either, so I'm not sure if that fix is correct.

**Comment by joernheissler** [comment]
I looked into it.
```python
from copy import deepcopy, copy
from asn1crypto.util import timedelta, timezone
copy(timezone(timedelta()))
```
fails.

The proper fix appears to be adding a [`__getinitargs__`](https://github.com/python/cpython/blob/fa919fdf2583bdfead1df00e842f24f30b2a34bf/Lib/datetime.py#L2206-L2210) method. `copy` calls some pickle functions which calls [`__reduce__`](https://github.com/python/cpython/blob/fa919fdf2583bdfead1df00e842f24f30b2a34bf/Lib/datetime.py#L1193) which calls `__getinitargs__`.

Still I'm curious how you trigger the problem from asn1crypto.

**Comment by luckylud** [comment]
I've spend some time to catch the cause of this problem, it's with **core.UTCTime** class
After get **.native** attr of an **UTCTime** object, when calling untag() a TypeError exception is raised during deepcopy. (Only python 2.x )

Here come the test to reproduce the error:
```
a = core.UTCTime(datetime.now(tz=util.timezone.utc))
a.untag()
a.native
a.untag()
```
The first call to a.untag() is ok, but not the second.

**Comment by joernheissler** [comment]
Ah, I forgot the `.native`. All makes sense now, thanks!
It's not really related to `untag` but to copying in general.

Hope you don't mind that I closed your PR and added a different one?

**Comment by luckylud** [comment]
No problem your fix is the right way to solve this issue, better than my quick fix or drop python2 support.

**Comment by wbond** [comment]
The fix was part of the 1.3.0 release
